src/kemal/param_parser.cr

```diff
@@ -60,6 +66,17 @@ class Kemal::ParamParser
     end
   end
 
+  def parse_msgpack
+    body = @request.body as String
+    unpacker = MessagePack::Unpacker.new(body.bytes)
+    case data = unpacker.read
+    when Hash
+      data.each do |k, v|
+        @params[k as String] = v as AllParamTypes
```

Comment:
yes, I do not know why `AllParamTypes` does not include `UInt8`
the **author** must have a reason :stuck_out_tongue_winking_eye: